hid-uclogic-rdescs.h

```diff
@@ -0,0 +1,76 @@
+#ifndef UCLOGIC_REPORT_DESCRIPTORS
+#define UCLOGIC_REPORT_DESCRIPTORS
+
+#include <linux/device.h>
```

Comment:
Hmm, is there are reason to include this here? Nothing in this file seems to be using it.